src/test/java/uk/gov/companieshouse/service/AppealServiceTest.java

```diff
@@ -33,12 +41,27 @@
     private static final String TEST_REASON_TITLE = "This is a title";
     private static final String TEST_REASON_DESCRIPTION = "This is a description";
 
+    private final ObjectMapper mapper = new ObjectMapper();
+    private List<Attachment> testAttachments;
+
     @InjectMocks
     private AppealService appealService;
 
     @Mock
     private AppealRepository appealRepository;
 
+    @Before
+    public void beforeTest() throws IOException {
+        List<Object> listOfAttachmentObjects = mapper.readValue(
+            new File("src/test/resources/data/listOfValidAttachments.json"), 
+            List.class
+        );
+                
+        testAttachments = listOfAttachmentObjects.stream()
```

Comment:
There is no need to map it twice, one time to HashMap and second time to Attachment.

It can be simplified by having:

```
testAttachments = mapper.readValue(
            new File("src/test/resources/data/listOfValidAttachments.json"),
            new TypeReference<>() {}
        );
```

Magic happens when `TypeReference` is used, it instructs object mapper what destination type to use.

Reply:
I'm going to read up on this magic